### PR TITLE
workspaces-applet: Really fix panel crash (issue #302)

### PR DESCRIPTION
**Pull request by City-busz**
This reverts commit 9422bb1563ac42759087b64d531898d567748b1e and add the set_size_request line to prevent the crash.

### CODE REVIEW

**Comment by ikeydoherty** [comment]
What I don't understand is why this only happens on Arch Linux :/ Theory would have me believe this is a 3.20 regression.

File: panel/applets/workspaces/WorkspacesApplet.vala [hunk 1]
```diff
@@ -21,29 +21,12 @@ public class WorkspacesApplet : Budgie.Applet
...
-    public void maybe_integrate_pager()
-    {
-        if (widget != null) {
-            return;
-        }
-
         widget = new Wnck.Pager();
-
-        box.add(widget);
+        widget.set_size_request (0, 0);
```
Comment (ikeydoherty) [inline panel/applets/workspaces/WorkspacesApplet.vala]: Drop the space before `(` and I'll merge it, just force push the change

**Comment by City-busz** [comment]
I don't understand too.

I dropped the space.

**Comment by ikeydoherty** [comment]
Odd stuff but we'll get it merged all the same, thanks :)

**Pull request merged**